src/user-management/interfaces/create-organization-membership-options.interface.ts

```diff
@@ -1,9 +1,13 @@
+import { RoleResponse } from "./role.interface";
+
 export interface CreateOrganizationMembershipOptions {
```

Comment:
We should also update the serializers: https://github.com/workos/workos-node/blob/208414a729d0a191e6872f288560553b3b8b4eb8/src/user-management/serializers/create-organization-membership-options.serializer.ts#L6-L11

Reply:
How `roleSlug` will get transformed into `RoleResponse`?